### PR TITLE
Updated Script to handle new excel layout

### LINKED ISSUE #2: Update Script to Handle New Excel Layout
### Description:

The script's current conversion process is based on an older Excel layout. With the recent changes to the Excel file structure, we must modify it to correctly proccess and convert the new Excel layout.

- **New Layout**: [ROOMz 20231.xls](https://github.com/GDSC-YU/excel-rooms-converter/files/12910031/ROOMz.20231.xls)
  - This will be the standard layout going forward.
  - The script will only need to be run once per semester.

**Note**: Feel free to rewrite the script in another language. Create a new branch, delete the existing content, and start from scratch.

### PR DESCRIPTION
**Pull request by Abd-404**
- Wrote the script using java
- Output is available in output.txt file
- Created a new Excel file, Rooms_copied.xlsx, which contains all the contents of original file(CtrlA + CtrlV :) ) to address issues with original file.
- Fixes #2

### CODE REVIEW

File: src/main/Main.java [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+		String courseName = "courseName";
+	
+
+		
+		String excelFilePath = ".\\Rooms_copied.xlsx";
+		FileInputStream inputStream = new FileInputStream(excelFilePath);
+		XSSFWorkbook workbook = new XSSFWorkbook(inputStream);
+		XSSFSheet sheet = workbook.getSheetAt(0);
+		CellAddress firstRoomCell = new CellAddress("Y4");
+		List<String> daysList = List.of("Sunday", "Monday", "Tuesday", "Wednesday", "Thursday");
```
Comment (g9wz) [inline src/main/Main.java:27]: to lowercase

File: src/main/Main.java [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+					int indexOfStartColon = courseTimeStart.indexOf(':');
+					String startHourString = courseTimeStart.substring(0, indexOfStartColon);
+					String startMinString = courseTimeStart.substring(indexOfStartColon + 1, indexOfStartColon + 3);
+					int startHour = convertTOInt(startHourString);
+					int startMin = convertTOInt(startMinString);
+					if (courseTimeStart.endsWith("PM") && startHour != 12) {
+						startHour += 12;
+					}
+					LinkedHashMap<String, String> startTimesMap = new LinkedHashMap<>(Map.of("hour", ";"+startHour+";"));
+					startTimesMap.put("min", ";"+startMin+";");
```
Comment (g9wz) [inline src/main/Main.java:78]: minute instead of min

File: src/main/Main.java [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+					int indexOfEndColon = courseTimeEnd.indexOf(':');
+					String endHourString = courseTimeEnd.substring(0, indexOfEndColon);
+					String endMinString = courseTimeEnd.substring(indexOfEndColon + 1, indexOfEndColon + 3);
+					int endHour = convertTOInt(endHourString);
+					int endMin = convertTOInt(endMinString);
+					if (courseTimeEnd.endsWith("PM") && endHour != 12) {
+						endHour += 12;
+					}
+					LinkedHashMap<String, String> endTimesMap = new LinkedHashMap<>(Map.of("hour", ";"+endHour+";"));
+					endTimesMap.put("min", ";"+endMin+";");
```
Comment (g9wz) [inline src/main/Main.java:91]: minute instead of min

**Review by g9wz — CHANGES_REQUESTED** [review]
kindly resolve conflicts too

**Comment by Abd-404** [comment]
done

**Review by g9wz — APPROVED** [review]
LGTM

**Pull request merged**